app-localize-behavior.js

```diff
@@ -284,20 +282,16 @@ export const AppLocalizeBehavior = {
         proto.__localizationCache.messages[messageKey] = translatedMessage;
       }
 
-      var args = {};
-      for (var i = 1; i < arguments.length; i += 2) {
-        args[arguments[i]] = arguments[i + 1];
-      }
-
+      const replacements = args.reduce((acc, a, idx) => idx % 2 ? acc : { ...acc, [args[idx - 1]]: a }, {});
```

Comment:
Not opposed to putting this back to a `for` if this is too unreadable. I just like things like this being isolated as much as possible.